src/SpectatorView.Unity/Assets/SpectatorView.Editor/Scripts/CompositorWindow.cs

```diff
@@ -184,6 +184,13 @@ private void CompositeGUI()
 
                             compositionManager.OutputDevice = supportedOutputDevices[selectedIndex];
 
+                            EditorGUILayout.EndHorizontal();
+                            EditorGUILayout.Space();
+                            EditorGUILayout.BeginHorizontal();
+
+                            var lowLatencyLabel = new GUIContent("Low Latency Mode", "Disables buffers to prevent stutters on performance or IO problems. Enable only for live production.");
+                            compositionManager.LowLatencyMode = EditorGUILayout.Toggle(lowLatencyLabel, compositionManager.LowLatencyMode);
```

Comment:
Does this only impact black magic output streams or does this also impact the frame shown in the compositor? If this only impacts black magic playback, it seems like we may want this property hidden behind IFrameProvider::SupportsOutput

Reply:
The low latency mode also reduces the size of the incoming frame queue. 
If there is a probable use-case where the output via the preview window would need to be low-latency hiding the switch/slider might spark some confusion?

To prevent this property to be too prominent towards the selection of the frame providers we might also move it in the UI below "Frame time adjustment" where it might fit better.